queries.sql

```diff
@@ -68,4 +68,33 @@ SELECT AVG(weight_kg) FROM animals;
 --  What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
 
- 
\ No newline at end of file
+ 
+                ------ day 3 ---------
+
+SELECT name, full_name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+where full_name = 'Melody Pond';
+SELECT animals.name AS animal_name, species.name AS species FROM animals
+JOIN species ON animals.species_id = species.id  
+where species.name = 'Pokemon';
+
+SELECT full_name, name FROM owners
+left JOIN animals ON owners.id = animals.owner_id;
+SELECT species.name, count(species_id) AS species_count FROM species
+JOIN animals ON species.id = animals.species_id 
+GROUP BY species.name;
+
+SELECT species.name AS species, full_name, animals.name AS animal_name FROM animals 
+JOIN species ON animals.id = species.id 
+JOIN owners ON animals.id = owners.id 
+where full_name = 'Jennifer Orwell'  AND species.name = 'Digimon';
+
```

Comment:
- [ ] Good job updating the query but you missed updating the screenshot in the pull request. Please make sure to update the screenshot in the pull request with the updated query as requested by the previous reviewer.

Reply:
Thanks for the comment, but the screenshot is already updated.

Reply:
Oops 😔. I added the screenshot but did not click on the update button to permanently save, I just previewed the changes.
By any way thanks for your review.